onnx_tf/handlers/backend/onehot.py

```diff
@@ -38,7 +38,7 @@ def version_9(cls, node, **kwargs):
     return [
         cls.make_tensor_from_onnx_node(
             node,
-            inputs=[indices, depth[0], on_value, off_value],
+            inputs=[indices, depth, on_value, off_value],
```

Comment:
The change looks right to me. But just curious, any reason for this change? Did you get any trouble form using the code before this change?

Reply:
this was onehot.py was some issue with rebasing before. I didn't work on it. Now the commits should be in order and onehot.py is not part of this PR.